test/models/movie.test.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const Movie = require('../../lib/models/movie');
+
+describe('movie model', () => {
+
+  describe('serialize', () => {
+
+    it('includes all of the necessary fields', () => {
+      return 'no coverage';
+    });
```

Comment:
Was I supposed to remove this when I introduced lines 13 - 22? @imWillX

Reply:
This doesn't do anything so yes, this can be removed.